pulpcore/cli/container/repository.py

```diff
@@ -286,3 +287,21 @@ def copy_manifest(
         digests=digests or None,
         media_types=media_types or None,
     )
+
+
+@repository.command(allowed_with_contexts=push_container_context)
```

Comment:
Does the `allowed_with_contexts` parameter have an impact on calling remove_image for regular container repositories?

Reply:
Yes, the command doesn't show up if the context is regular container repositories. I don't see a remove_image call for regular container repositories.